quickwit/quickwit-serve/src/index_api/rest_handler.rs

```diff
@@ -141,6 +146,129 @@ fn get_indexes_metadatas_handler(
         .map(make_response)
 }
 
+
+
+// This structure represents describe Index
+
+#[derive(Serialize, Deserialize)]
```

Comment:
can you add `utoipa::ToSchema`? It's for the OpenApi stuff.

You will need to add `IndexStats` in `components(schemas(ToggleSource, SplitsForDeletion, IndexStats))`.